sos/report/plugins/ovn_central.py

```diff
@@ -25,6 +25,20 @@ class OVNCentral(Plugin):
     profiles = ('network', 'virt')
     containers = ('ovn-dbs-bundle.*', 'ovn_cluster_north_db_server')
 
+    def _find_ovn_controller_sock(self):
+        if self._container_name:
+            cmd = "find %s -name %s" % (self.ovn_controller_sock_path,
+                                        self.ovn_controller_sock_regex)
+            res = self.exec_cmd(cmd, timeout=None, foreground=True,
+                                container=self._container_name)
```

Comment:
Use `stat` here - we aren't guaranteed to have `find` available in a given container, whereas `stat` is from `coreutils` which is much more common.

```python
_sfile = os.path.join(self.ovn_controller_sock_path,
                      self.ovn_controller_sock_regex)
res = self.exec_cmd("stat -t %s" % _sfile",
                    container=self._container_name)
[...]
```

Reply:
Good point, I make some test, but stat not accept the * included in the self.ovn_controller_sock_regex, when I test on shell the command expand the '*' but running in code the output looks as:

`stat: cannot statx '/var/run/openvswitch/ovn-controller.*.ctl': No such file or directory`

So taking your tip about use a command on coreutils I move the solution to ls + pattern matching over the regex.